packages/backend/src/backend_factory.test.ts

```diff
@@ -9,8 +9,8 @@ import { Bucket } from 'aws-cdk-lib/aws-s3';
 import { BackendFactory } from './backend_factory.js';
 import { App, Stack } from 'aws-cdk-lib';
 import { Template } from 'aws-cdk-lib/assertions';
-import assert from 'node:assert';
 import { ClientConfig } from '@aws-amplify/client-config';
+import assert from 'node:assert';
```

Comment:
since we're not changing `backend` package please revert this and remove `backend` from changesets.